js/atree_constants.js

```diff
@@ -7956,8 +7956,8 @@ const atrees = {
                 "Purification"
             ],
```

Comment:
haha prolly its this issue

`_parents` tag is fake, it isn't read (it was a legacy tag, for things that look like they connect ingame but actually don't)

i would just delete `_parents` and fix `parents`

Reply:
uh ok
idk what u mean but ill try it